src/app/technical-record/vehicle-summary/edit/type-approval/type-approval.component.html

```diff
@@ -0,0 +1,80 @@
+<ng-container>
```

Comment:
This component is not used. The Type Approval section is added directly on the Vehicle Summary component.

Reply:
we have it here vehicle-summary/edit/vehicle-summary-edit.component.html `line 357`